.circleci/config.yml

```diff
@@ -157,6 +156,33 @@ jobs: # a collection of steps
       - run:
           name: build
           command: yarn sample build
+      - store_artifacts:
+          path: js-miniapp-sample/build
+          prefix: build
+
+  deploy-sample:
+    working_directory: ~/js-miniapp-sample
+    docker:
+      - image: circleci/node:10.16.3
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ./
+      - run:
+          name: Deploy Sample App to Github Pages
+          command: |
+            set -e
+            git checkout gh-pages
+            cp -r ./js-miniapp-sample/build/ ./sample
+            if [[ `git status sample --porcelain` ]]; then
+              git add sample
+              git config user.name "CI Publisher"
+              git config user.email "dev-opensource@mail.rakuten.com"
+              git commit -m "sample: Deploy JS Sample App [ci skip]"
+              git push origin gh-pages
+            else
+              echo "Nothing to commit"
+            fi
 
   publish-sdk:
```

Comment:
Now our build & publish flow is sdk -> bridge -> sample. Since there are some object models being exported from js bridge so I think the flow should be bridge -> sdk -> sample.

Reply:
I think that the jobs will run in parallel, so the order in the config file just determines what order they show in the UI on CircleCI. But I noticed that the jobs weren't really grouped together, so I rearranged them a to follow bridge -> sdk -> sample.